src/core/effects/qgsblureffect.cpp

```diff
@@ -44,14 +45,16 @@ void QgsBlurEffect::draw( QgsRenderContext &context )
 
 void QgsBlurEffect::drawStackBlur( QgsRenderContext &context )
 {
+  double blurLevel = context.convertToPainterUnits( mBlurLevel, mBlurUnit, mBlurMapUnitScale );
```

Comment:
again, may as well round this to an int value
```suggestion
  int blurLevel = std::round( context.convertToPainterUnits( mBlurLevel, mBlurUnit, mBlurMapUnitScale ) );
```